web/client/epics/catalog.js

```diff
@@ -467,7 +467,9 @@ export default (API) => ({
 
                         const metadataFlow = Rx.Observable.defer(() => axios.get(metadataUrl, {headers: {'Accept': 'application/xml'}}))
                             .pluck('data')
-                            .map(metadataXml => new DOMParser().parseFromString(metadataXml))
+                            .map(metadataXml => {
+                                return (new DOMParser()).parseFromString(metadataXml, "text/xml");
```

Comment:
The import seems still in the file, is it correct?
https://github.com/geosolutions-it/MapStore2/blob/bf41f19b588d6cd3becc13922f1dcd3cef56d05e/web/client/epics/catalog.js#L12